src/macro.ts

```diff
@@ -130,15 +129,14 @@ export default class Macro extends StrictMacro {
   hardKill(): this {
     if (myClass() === $class`Grey Goo`) return this;
 
-    const [bestKillSpell, otherKillSpell] =
-      myClass() === $class`Pastamancer`
-        ? $skills`Weapon of the Pastalord, Saucegeyser`
-        : $skills`Saucegeyser, Weapon of the Pastalord`;
     return this.externalIf(
       myPrimestat() === $stat`mysticality`,
       Macro.tryHaveSkill($skill`Stuffed Mortar Shell`)
-        .trySkillRepeat(bestKillSpell)
-        .trySkillRepeat(otherKillSpell)
+        .externalIf(
+          myClass() === $class`Pastamancer` && have($skill`Fearful Fettucini`),
+          Macro.trySkillRepeat($skill`Fearful Fettucini`)
+        )
+        .trySkillRepeat($skill`Saucegeyser`)
```

Comment:
only do this if we're adding a lantern of some sort as well. also, fearful is only better if you have and are equipping the velour veil, so make that a condition here too

Reply:
I think it is unlikely we will be adding velour veil or lanterns, but who could know